wagtail/wagtailadmin/static/wagtailadmin/css/.gitignore

```diff
@@ -0,0 +1,9 @@
+core.css
+layouts/home.css
+layouts/login.css
+layouts/page-editor.css
+layouts/preview.css
+normalize.css
+panels/rich-text.css
+userbar.css
+userbar_embed.css
```

Comment:
I was going for minimal changes in this PR, but if the devs are happy with this, it would certainly simplify the `.gitignore` rules.